v0/destinations/am/utils.js

```diff
@@ -69,11 +71,31 @@ function getBrand(payload, sourceKey, Config) {
   return undefined;
 }
 
+function getEventId(payload) {
+  if (isDefinedAndNotNull(payload?.properties?.event_id)) {
```

Comment:
rather than referencing `payload?.properties?.event_id`, `payload?.context?.traits?.event_id`, and `payload?.traits?.event_id` multiple times store them in a variable and do the next steps makes code easier to understand

Reply:
Done